tol_lab_share/messages/mappers/traction_to_warehouse.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+import logging
+
+from tol_lab_share.constants.input_traction_volume_tracking_message import TRACTION_LIMS
+from tol_lab_share.messages.mlwh.create_aliquot_message import CreateAliquotInWarehouseMessage
+from tol_lab_share.messages.consumed import TractionToWarehouseMessage
+
+logger = logging.getLogger(__name__)
+
+
+class TractionToWarehouseMapper:
```

Comment:
Sorry I should have also commented about the name of this class.  We should make it more specific to the use of this message to avoid future naming complications.